scripts/leaderboard.js

```diff
@@ -0,0 +1,146 @@
+require("dotenv").config();
+
```

Comment:
I understand this is a javascript version of the file above, same review applies.

With the added question: _can't we re-use code?!_